src/canvas/graphic.ts

```diff
@@ -158,15 +159,25 @@ function brushPath(ctx: CanvasRenderingContext2D, el: Path, style: PathStyleProp
         }
     }
 
-    const lineDash = style.lineDash;
-    const lineDashOffset = style.lineDashOffset;
+    let lineDash = style.lineDash;
+    let lineDashOffset = style.lineDashOffset;
 
     const ctxLineDash = !!ctx.setLineDash;
 
     // Update path sx, sy
     const scale = el.getGlobalScale();
     path.setScale(scale[0], scale[1], el.segmentIgnoreThreshold);
 
+    if (lineDash) {
+        const lineScale = (style.strokeNoScale && el.getLineScale) ? el.getLineScale() : 1;
+        if (lineScale && lineScale !== 1) {
```

Comment:
@pissang Suddenly I remember a previous doubt about the value of `Transformable.getLineScale()`, that is, is it possible to be `0`? If possible, maybe we should set `lineDash` as an empty array and `lineDashOffset` as `0` to avoid the calculation?

Reply:
I found it's possible when the width is `0`. It won't have any effect even if we don't set `lineDash` as an empty array and `lineDashOffset` as 0 since this element is invisible for its stroke width is `0`. So, there may be no need to do it.